### PR TITLE
Update code to current LTS version

### PR DESCRIPTION
**Pull request by jacek-foremski**
> [EZP-27160](http://jira.ez.no/browse/EZP-27160)
> [EZP-27098](http://jira.ez.no/browse/EZP-27098)
#
My goal was to: 
- fix any errors that prevented the code from running in the last LTS
- fix unit tests and make them pass
- change the codebase to follow PSR-2 and PHP/Symfony best practices
- remove unnecessary code

### CODE REVIEW

File: Command/CreateTweetContentCommand.php
Comment (bdunogier) [inline Command/CreateTweetContentCommand.php]: Not related to this line, but there's a typo in this change's commit message: "Remove call to *depracated* method setCurrentUser".

File: eZ/Publish/FieldType/Tweet/Type.php
Comment (bdunogier) [inline eZ/Publish/FieldType/Tweet/Type.php]: Nitpick: single quotes :)

File: eZ/Publish/FieldType/Tweet/Type.php
Comment (bdunogier) [inline eZ/Publish/FieldType/Tweet/Type.php]: Naming: i'd change this to `isAuthorApproved` (verb).

File: eZ/Publish/FieldType/Tweet/Type.php
Comment (bdunogier) [inline eZ/Publish/FieldType/Tweet/Type.php]: Short array syntax while at it ?

File: eZ/Publish/FieldType/Tweet/Type.php
Comment (bdunogier) [inline eZ/Publish/FieldType/Tweet/Type.php]: CS: space before return.

File: Resources/config/fieldtypes.yml
Comment (bdunogier) [inline Resources/config/fieldtypes.yml]: Unless I'm mistaken, the quotes aren't necessary (see the examples from http://symfony.com/doc/current/service_container.html). I'd remove them altogether.

**Review by bdunogier — CHANGES_REQUESTED** [review]
A couple minor remarks, but overall: good job !

**Comment by bdunogier** [comment]
Note that some of my comments show as outdated, as I have reviewed per commit. Don't forget to open each at least once before dismissing them :)

For the future, the CS change on the whole repository should maybe be done first, as it made it very hard to distinguish CS changes from code changes when reviewing the PR as a whole.

**Comment by jacek-foremski** [comment]
I made requested changes and few more :)

I agree with your comment about CS changes and I will do just that next time.

**Review by bdunogier — APPROVED** [review]

**Comment by bdunogier** [comment]
@natanael89 can you rebase on top of master ? A typo fix ("ocurred" / "occurred") was merged yesterday that conflicts with your CS changes.

Once that is done, we can merge it.

**Comment by jacek-foremski** [comment]
@bdunogier I rebased it, so it's ready to be merged.

File: DependencyInjection/EzSystemsTweetFieldTypeExtension.php
Comment (lolautruche) [inline DependencyInjection/EzSystemsTweetFieldTypeExtension.php]: `Yaml::parse()` must take YAML content now:

```php
$config = Yaml::parse(file_get_contents(__DIR__ . '/../Resources/config/ez_field_templates.yml'));
```

File: Resources/views/fields/eztweet.html.twig
Comment (lolautruche) [inline Resources/views/fields/eztweet.html.twig]: Missing empty line at end of file

File: Resources/views/platformui/content_type/view/eztweet.html.twig
Comment (lolautruche) [inline Resources/views/platformui/content_type/view/eztweet.html.twig]: Missing empty line

**Review by lolautruche — APPROVED** [review]
Besides my comment about YAML loading and minor stuff, +1

**Comment by bdunogier** [comment]
We will merge once @lolautruche's suggestions have been integrated. Thank you !

**Comment by jacek-foremski** [comment]
Thank you for your comments @lolautruche! Your suggestions have been added.

**Review by andrerom — APPROVED** [review]

**Comment by mere-teresa** [comment]
Thank you @natanael89 !

**Pull request merged**
